src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/alert-title-template.mdx

```diff
@@ -0,0 +1,102 @@
+tags:
+  - Alerts and applied intelligence
+  - Alerts
+  - Alert conditions
+metaDescription: 'Give your alert condition a unique title to easily identify related incidents.'
+redirects:
+  - /docs/alerts/new-relic-alerts/defining-conditions/alert-condition-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions
+  - /docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-violation-descriptions/
+freshnessValidatedDate: never
+---
+
+import alertsAddingAlertDetailstoanAlertCondition from 'images/alerts_screenshot-crop_adding-alert-details-to-an-alert-condition.webp'
+
+
+A title template is used when incidents are opened by the condition. It overrides the default title.
+
+## Why use a title template?
+An alert condition is the foundation for incident creation. Assigning a meaningful title to your alert condition ensures you can quickly pinpoint the source and specific parameters that triggered an incident. Whether you're using our guided mode or crafting a condition from a chart or policy, leverage our title template in the final step to customize incident titles for easy identification.
+
+<img
+    title="alert details page"
+    alt="A screenshot depicting the final step of creating an alert condition- the alert details page"
+    src={alertsAddingAlertDetailstoanAlertCondition}
+/>
+
+<figcaption>
+    Use the title template to easily identify related incidents.
+</figcaption>
+
+## Before you begin [#begin]
+
+Here are some key things to note before you use our title template: 
+
+* **Compatibility**: The title template feature is available for NRQL conditions only.
+* **Character limit**: The maximum length for a title template is 256 characters.
+* **Formatting**: The title template must follow a valid [Handlebars](https://handlebarsjs.com/guide/) format. 
+* **Lost signal incidents**: For incidents triggered by a lost signal, the default title will always be used. To learn more about signal loss, see [Loss of signal and gap filling](/docs/apis/nerdgraph/examples/nerdgraph-api-loss-signal-gap-filling/). 
+
+## Create a title [#example]
+
+Let's say you run an e-commerce site and you have created an alert condition to trigger an incident whenever your site generates 200-response errors. You can use a title template to make incidents from this condition easily identifiable.
+
+You have named this alert condition `high error rate - checkout`. But you want the incident title to reflect where the errors are being generated. In this case, the errors for this incident are in your checkout.
+
+**Example**: 
+
+```handlebars
+{{conditionName}} targeting {{targetName}} incident
+```
+
+This template would generate the following incident title:
+
+```
+High error rate- checkout targeting `checkout` incident
```

Comment:
I appreciate attempting to provide a real world example here, but `checkout` is in the condition name, so adding the `{{targetName}}` to the title template seems unnecessary. I would change the condition name to just `high error rate` or `high error rate - production` or something like that so it makes sense why the user would want to include `{{targetName}}` in the title template.

Reply:
Thanks for this! I edited this.